### PR TITLE
Fix calendar render

### PR DESCRIPTION
**Pull request by neoformit**
Fixes https://github.com/galaxyproject/tiaas2/issues/52 by replacing newlines with spaceless `<br>` tags when rendering event descriptions. 

Also switched Vue CDN link to a prod release.

Also fixes the issue of calendar panel months being wrong:

image

### CODE REVIEW

File: training/templates/training/calendar.html [hunk 1]
```diff
@@ -324,7 +329,9 @@ <h4 style="text-align: center; margin-top: 50px;">
             title: "{{ event.title }}",
```
Comment (hexylena) [inline training/templates/training/calendar.html:329]: should we replace this with a proper API that returns a real proper safely formatted json blob? This seems like an accident waiting to happen again (we had the same problem in the GTN, constantly fighting until we replaced it with a proper json serialiser.)

I'm happy to take the serializer portion if you can refactor the vue bits for an API at some point? I.e. **not today**, a friday night, stop answering my work mails @neoformit and enjoy your weekend mate.

**Review by hexylena — APPROVED** [review]
Let's try this for now

File: training/templates/training/calendar.html (hunk 1 above)
Comment (neoformit) [reply training/templates/training/calendar.html:329]: 😛 that sounds like a good idea, I'm happy to do the Vue end. But feel free to merge this as a patch in the meantime if you like.

Comment (hexylena) [reply training/templates/training/calendar.html:329]: defo!

**Pull request merged**